CMakeLists.txt

```diff
@@ -16,7 +18,7 @@ option(USE_NATIVE_INSTRUCTIONS "USE_NATIVE_INSTRUCTIONS makes rpcs3 compile with
 option(WITH_LLVM "Enable usage of LLVM library" ON)
 option(BUILD_LLVM "Build LLVM from git submodule" OFF)
 option(STATIC_LINK_LLVM "Link against LLVM statically. This will get set to ON if you build LLVM from the submodule." OFF)
-option(USE_FAUDIO "FAudio audio backend" ON)
+option(USE_FAUDIO "FAudio audio backend" OFF)
```

Comment:
Why?

Reply:
reverted

Reply:
hmm. did you forget to submit it?

Reply:
I am in the process of fixing stuff, I will submit when I am finished.